### PR TITLE
fix usage around find_package

### PR DESCRIPTION
**Pull request by prince-chrismc**

### CODE REVIEW

File: recipes/xerces-c/all/conanfile.py [hunk 1]
```diff
@@ -12,8 +13,8 @@ class XercesCConan(ConanFile):
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "http://xerces.apache.org/xerces-c/index.html"
     license = "Apache-2.0"
-    exports_sources = ["CMakeLists.txt"]
-    generators = "cmake"
+    exports_sources = ["CMakeLists.txt", "patches/*"]
+    generators = "cmake", "cmake_find_package_multi"
```
Comment (prince-chrismc) [inline recipes/xerces-c/all/conanfile.py:17]: this

File: recipes/xerces-c/all/patches/0001-usercurl-cmake-targets.patch [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
+index 344851fad..f396fc5ef 100644
+--- a/src/CMakeLists.txt
++++ b/src/CMakeLists.txt
+@@ -1085,7 +1085,7 @@ endif()
+ if(XERCES_USE_NETACCESSOR_CURL)
+   list(APPEND libxerces_c_SOURCES ${curl_sources})
+   list(APPEND libxerces_c_HEADERS ${curl_headers})
+-  list(APPEND libxerces_c_DEPS ${CURL_LIBRARIES})
++  list(APPEND libxerces_c_DEPS CURL::libcurl)
```
Comment (prince-chrismc) [inline recipes/xerces-c/all/patches/0001-usercurl-cmake-targets.patch:21]: and this

**Review by prince-chrismc — COMMENTED** [review]
The magic is

**Comment by FMeinicke** [comment]
Thanks!

**Pull request merged**